src/components/core/Sandtank/template.html

```diff
@@ -59,8 +59,6 @@
             @click="runSimulation"
             :loading="simulationLoading"
             color="primary"
-            :outlined="!needRunSimulation"
-            :disabled="!needRunSimulation"
```

Comment:
Why do you remove those if you made SIM_RUN_NEEDED respect the initial run part?

Reply:
I also considered leaving them in, but might just check with Lisa and Jill if they want the Run Simulation button to always be enabled or just to be enabled in the initial run case.